webpack/module-minifier-plugin-5/src/index.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+export * from './Constants';
+export * from './GenerateLicenseFileForAsset';
+export * from './ModuleMinifierPlugin.types';
+export * from './ModuleMinifierPlugin';
+export type {
+  ILocalMinifierOptions,
+  IMinifierConnection,
+  IModuleMinificationCallback,
+  IModuleMinificationErrorResult,
+  IModuleMinificationRequest,
+  IModuleMinificationResult,
+  IModuleMinificationSuccessResult,
+  IModuleMinifier,
+  IModuleMinifierFunction,
+  IWorkerPoolMinifierOptions
+} from '@rushstack/module-minifier';
+export {
+  getIdentifier,
+  LocalMinifier,
+  MessagePortMinifier,
+  NoopMinifier,
+  WorkerPoolMinifier
+} from '@rushstack/module-minifier';
```

Comment:
I get exporting these from the v4 index for backwards compatibility. In V5, should we instead just have consumers get these from the packages directly?